content/en/docs/releases/supported-releases/index.md

```diff
@@ -66,6 +73,8 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
 | 1.7             | No                   | August 21, 2020   | Feb 25, 2021             | 1.16, 1.17, 1.18              | 1.15                               |
 | 1.6 and earlier | No                   |                   |                          |                               |                                    |
 
+The intent of the project is to support the last 4 versions of Kubernetes with each Istio release; however, this may change on a per-version basis.
```

Comment:
```suggestion
The intent of the project is to support the last four versions of Kubernetes with each Istio release; however, this may change on a per-version basis.
```

Reply:
I'm not sure if we want to add some additional detail on support for a new version of Kubernetes as it comes out. 1.26.x being released after 1.16 was shipped in this case. There was discussion in a TOC meeting about testing requirements to claim that the release is supported, and not just tested.

Reply:
My 2c - "The project targets supporting four versions of Kubernetes each release, but based on release schedules this may not be met, nor is it guaranteed these will be the for *most recent* versions"

Or something along those lines?

I agree its problematic to say its "last 4" versions

Reply:
Updated